tests/conftest.py

```diff
@@ -80,8 +101,9 @@ def datacite_record_partial():
 @pytest.fixture()
 def timdex_record_required_fields():
     return TimdexRecord(
+        citation="Creator (PubYear): Title. Publisher. (resourceTypeGeneral). ID",
```

Comment:
See note about citation, we shouldn't include it in the required TIMDEX fields record.